pkg/cmd/target/control-plane.go

```diff
@@ -23,11 +23,15 @@ func NewCmdTargetControlPlane(f util.Factory, ioStreams util.IOStreams) *cobra.C
 	}
 	cmd := &cobra.Command{
 		Use:   "control-plane",
-		Short: "Set control plane for next operations",
-		Example: `# target control-plane for current shoot cluster
-gardenctl target control-plane`,
+		Short: "Target the control plane of the shoot",
+		Long:  "Target the control plane of the shoot cluster to set the scope for the next operations",
+		Example: `# target control-plane of currently selected shoot cluster
+gardenctl target control-plane
+
+# target control-plane of shoot my-shoot
+gardenctl target control-plane --shoot my-shoot --project my-project --garden my-garden`,
```

Comment:
```suggestion
gardenctl target control-plane --garden my-garden --project my-project --shoot my-shoot`,
```
turn it around, otherwise you will have a hard time with the completions

Reply:
ok